backup_and_restore/replacing-unhealthy-etcd-member.adoc

```diff
@@ -0,0 +1,37 @@
+[id="replacing-unhealthy-etcd-member"]
+= Replacing an unhealthy etcd member
+include::modules/common-attributes.adoc[]
+:context: replacing-unhealthy-etcd-member
+
+toc::[]
+
+This document describes the process to replace a single unhealthy etcd member.
+
+The process to replace the unhealthy member depends on whether the etcd member is unhealthy because the machine is not running or the node is not ready, or whether it is unhealthy because the etcd Pod is crashlooping.
```

Comment:
This sentence is long; can you break it up or pare it down? Maybe something like:
```
The process to replace the unhealthy member depends on whether:
* The machine is not running or the node is not ready; or,
* The etcd Pod is crashlooping.
```
Or maybe:
`This process depends on whether the etcd member is unhealthy because the machine is not running or the node is not ready, or because the etcd Pod is crashlooping.`